### PR TITLE
[FIX] broken link

### PR DESCRIPTION
**Pull request by houtan-rocky**
Change broken link going to usage api that doesn't exist to usage  docs.

### CODE REVIEW

**Comment by houtan-rocky** [comment]
@kangax @znarf 
Hello,
Could you review this please?

**Comment by nicolo-ribaudo** [comment]
Could you link to https://babeljs.io/docs/babel-core instead?

**Comment by houtan-rocky** [comment]
@nicolo-ribaudo 
I changed the link and the text related to it, could you re-check please?

File: website/data/tools/node/usage.md [hunk 1]
```diff
@@ -6,6 +6,6 @@ require("@babel/core").transform("code", {
 
 <blockquote class="alert alert--info">
   <p>
-    For full documentation on the Babel API see the <a href="/docs/usage/api/">usage docs</a>.
+    For detailed insights and usage instructions on Babel Core, explore the <a href="/docs/babel-core">official Babel Core documentation</a>.
```
Comment (existentialism) [inline website/data/tools/node/usage.md:9]: I think we can simplify this to just:

```suggestion
    Explore the <a href="/docs/babel-core">@babel/core documentation</a> for the full API.
```

**Comment by houtan-rocky** [comment]
@nicolo-ribaudo Yeah that right thanks @:+1:

**Review by JLHwung — APPROVED** [review]

**Pull request merged**